databuilder/databuilder/extractor/base_postgres_metadata_extractor.py

```diff
@@ -36,7 +36,7 @@ class BasePostgresMetadataExtractor(Extractor):
     DEFAULT_CLUSTER_NAME = 'master'
 
     DEFAULT_CONFIG = ConfigFactory.from_dict(
-        {WHERE_CLAUSE_SUFFIX_KEY: ' ', CLUSTER_KEY: DEFAULT_CLUSTER_NAME, USE_CATALOG_AS_CLUSTER_NAME: True}
+        {WHERE_CLAUSE_SUFFIX_KEY: 'true', CLUSTER_KEY: DEFAULT_CLUSTER_NAME, USE_CATALOG_AS_CLUSTER_NAME: True}
```

Comment:
What does `WHERE_CLAUSE_SUFFIX_KEY: 'true'` do? Looking at the code it seems like this would get passed into this part of the query:
```sql 
WHERE att.attnum >=0 and {where_clause_suffix}
```

Reply:
it's a common placeholder for any additional (you can see also 1=1) conditions in sql query. This way you don't have to add `and` conditionally to where clause